### ISSUE TITLE
Segmentation fault when using over ssh

### ISSUE THREAD

**Issue by nealot** [kelseyhightower/nocode#2426]
Hi, love this project! I recently wanted to test this over ssh but recieved a segmentation fault error. Any suggestions on how to resolve?

**Comment by hydra007** [comment]
Don't.

**Comment by MininMobile** [comment]
Yeah, I've had this before and it's a really annoying bug I had to learn to fix the hard way...

Run the following in the terminal in the same directory as your app
```

```
Then run it again but instead of the ` ` argument use ` ` instead
```

```
Now, if you get this, you should be grand!
```

```
